dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumEventExt.kt

```diff
@@ -100,3 +101,71 @@ internal fun RumActionType.toSchemaType(): ActionEvent.Type1 {
         RumActionType.CUSTOM -> ActionEvent.Type1.CUSTOM
     }
 }
+
+internal fun NetworkInfo.toResourceConnectivity(): ResourceEvent.Connectivity {
```

Comment:
I didn't see the unit tests for those 2 extension methods.

Reply:
They are tested indirectly in the Resource / View scopes